stdlib/public/core/StringBridge.swift

```diff
@@ -250,11 +250,6 @@ public final class _NSContiguousString : _SwiftNativeNSString {
     }
   }
 
-  @objc
-  func _fastCharacterContents() -> UnsafeMutablePointer<UInt16>? {
-    return _core.elementWidth == 2 ? _core.startUTF16 : nil
-  }
-
   //
```

Comment:
Where is the proper place for these? It seems like someone chose the standard library, but you're moving it to the overlays. I'm unfamiliar with the reasoning behind either approach. If we move this to the overlays, why don't we move (almost) everything in this file too?